### DISCUSSION TITLE
Calculator Theme

### DISCUSSION THREAD

**Discussion by CarolineChiefari** [CarolineChief/Tafe240261#1] [General]
Hi guys,
Just wanted to let you know that the Main Page for the calculators is uploaded and I was debating whether to colour code each button with each individual calculator background colour? The Mortgage calculator im working on has a blue background so the button on the main page can be blue to match? Let me know what you think. 
Thanks

**Comment by Hayden-Badenhop** [comment]
I think that would work, the trip page will be a light grey, with a light grey button
